src/main/java/network/brightspots/rcv/CsvCvrReader.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * RCTab
+ * Copyright (c) 2017-2022 Bright Spots Developers.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+/*
+ * Purpose: Read and parse generic CSV files
```

Comment:
Nit: add period at end.

Reply:
I think it is, github is showing the "outdated" code here -- the "Files Changed" will show it truly resolved :)